deploy/runtools/firesim_topology_elements.py

```diff
@@ -526,6 +547,10 @@ def get_rootfs_name(self) -> Optional[str]:
             # cases
             return self.get_job_name() + "-" + rootfs_path.split("/")[-1]
 
+    def get_tar_name(self) -> str:
+        """ Get the name of the tarball on the run host"""
+        return "driver-bundle.tar.gz"
```

Comment:
Maybe this should be hoisted to the top of the file and put in all caps, per PEP8. I'll defer to @timsnyder